mtags/src/main/scala-3/scala/meta/internal/mtags/MtagsEnrichments.scala

```diff
@@ -121,4 +130,25 @@ object MtagsEnrichments extends CommonMtagsEnrichments:
     def backticked: String =
       KeywordWrapper.Scala3.backtickWrap(s)
 
+  extension (search: SymbolSearch)
+    def symbolDocumentation(symbol: Symbol)(using
+        Context
+    ): Option[SymbolDocumentation] =
+      def toSemanticdbSymbol(symbol: Symbol) =
+        SemanticdbSymbols.symbolName(
+          if !symbol.is(JavaDefined) && symbol.isPrimaryConstructor then
+            symbol.owner
+          else symbol
+        )
+      val sym = toSemanticdbSymbol(symbol)
+      val documentation = search.documentation(
+        sym,
+        new ParentSymbols:
+          def parents(): java.util.List[String] =
+            symbol.allOverriddenSymbols.map(toSemanticdbSymbol).toList.asJava
+      )
```

Comment:
optional - single method conversion
```suggestion
        () =>  symbol.allOverriddenSymbols.map(toSemanticdbSymbol).toList.asJava
```

Reply:
Good idea! Changed